include/onnxruntime/core/framework/framework_provider_common.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+struct OrtCustomOpDomain {
+  std::string domain_;
+  std::vector<const OrtCustomOp*> custom_ops_;
+};
+
```

Comment:
Previously I moved `OrtCustomOpDomain` to framwork_common.h but encountered following compile error:
warning C4099: 'onnxruntime::Model': type name first seen using 'struct' now seen using 'class'

The reason is the onnxruntime::Model is defined as struct in provider_api.h but defined as class in framework_common.h